src/Microsoft.IdentityModel.Protocols/Configuration/ConfigurationManager.cs

```diff
@@ -45,7 +45,7 @@ public class ConfigurationManager<T> : IConfigurationManager<T> where T : class
         /// <summary>
         /// 5 days is the default time interval that afterwards, <see cref="GetConfigurationAsync()"/> will obtain new configuration.
```

Comment:
Nice, the XML documentation is now incorrect... maybe it was worth updating it? :sweat_smile: